### DISCUSSION TITLE
桌面版的Alist挂载问题

### DISCUSSION THREAD

**Discussion by howdent** [AlistGo/alist#4769] [General]
好像是利用rclone挂载的，是不是可以添加自己设置挂载参数的功能？

**Comment by xhofe** [comment]
可以 下个版本添加

**Reply by howdent** [reply]
感谢添加了自定义参数的功能！
不过新版本好像有一点小问题，之前添加的自动挂载的条目，启动后就一直卡住转圈，无法卸载或删除，新添加的就没问题。
![image](https://github.com/alist-org/alist/assets/10974792/77007a4b-4c19-4719-bc24-3b2b4f8da3c0)

**Reply by xhofe** [reply]
已在3.22.1中修复

**Reply by BlueSkyXN** [reply]
无法卸载/删除的问题如何解决
目前是重装、删除配置文件、重启等操作均无效，这个配置是写在注册表了吗

**Reply by BlueSkyXN** [reply]
@xhofe xhofe
![https://image.3001.net/images/20231105/1699156401_654711b1a88a8aa674b12.png](https://image.3001.net/images/20231105/1699156401_654711b1a88a8aa674b12.png)

**Reply by xhofe** [reply]
> 无法卸载/删除的问题如何解决
> 目前是重装、删除配置文件、重启等操作均无效，这个配置是写在注册表了吗

我没理解你说的什么意思。

**Reply by BlueSkyXN** [reply]
> > 无法卸载/删除的问题如何解决
> > 目前是重装、删除配置文件、重启等操作均无效，这个配置是写在注册表了吗
> 
> 我没理解你说的什么意思。

如图所示，在不知道什么时候，我发现我的Alist Desktop的挂载页面变成那样，这个挂载记录目前无法删除、编辑、挂载、也没有日志，系统里面也没有这个挂载盘。
重装软件、删除配置文件后重装软件、重启软件、重启电脑等操作均无效。
而且这个记录始终保留，我不知道如何删除这个记录。
